server/src/domain/repositories/server-info.repository.ts

```diff
@@ -11,5 +11,5 @@ export interface GitHubRelease {
 export const IServerInfoRepository = 'IServerInfoRepository';
 
 export interface IServerInfoRepository {
-  getGitHubRelease(): Promise<GitHubRelease>;
+  getGithubRelease(): Promise<GitHubRelease>;
```

Comment:
Why did you rename this? It is called GitHub not Github.

Reply:
For the petty reason that the metric name was coming out as `get_git_hub_release` instead of `get_github_release`. But I can handle that in the decorator instead.

Reply:
I personally prefer `Github`. To me it boils down to the same discussion than API